src/Main.cpp

```diff
@@ -116,49 +117,64 @@ static void SurroundSelectionsWith(char ch1, char ch2) {
 	editor.EndUndoAction();
 }
 
+static char GetASCIICharacterFromKey(WPARAM wparam, LPARAM lparam) 
+{
+	BYTE keyboardState[256];
+	GetKeyboardState(keyboardState);
+
+	// https://msdn.microsoft.com/en-us/library/ms644984(v=VS.85).aspx
+	UINT scanCode = HIBYTE(LOWORD(lparam)); 
+	UINT virtualKeyCode = UINT(wparam);
+	UINT isMenuActive = 0;
+
+	WORD charResult = 0;
+
+	int isDeadKey = ToAscii(
+		virtualKeyCode,
+		scanCode,
+		keyboardState,
+		&charResult,
+		isMenuActive
+	);
+
+	return (CHAR)charResult;
```

Comment:
C++ style type-casting is preferred.
Cast to `char` as that is the return-type of the function.